rllib/execution/train_ops.py

```diff
@@ -55,13 +55,6 @@ def train_one_step(trainer, train_batch) -> Dict:
     trainer._counters[NUM_ENV_STEPS_TRAINED] += train_batch.count
     trainer._counters[NUM_AGENT_STEPS_TRAINED] += train_batch.agent_steps()
 
-    # Update weights - after learning on the local worker - on all remote
```

Comment:
actually why do we move this into Trainer?

Reply:
More logical to separate these two concepts:
- train one step
- update weights